src/functions/utils/utils.lua

```diff
@@ -91,34 +89,25 @@ function util.scandir(directory)
             return t
         end
 
-        local str = "ERROR: ERROR OCCURED!"
-        str:gsub(".", function(c) table.insert(t, c) end)
-
-        return t;
-    
+        str = util.ERROR_MESSAGE
     else
-        local t = {}
-        local str = "REQUIRES YOUR OWN IMPLEMENTATION!"
-        str:gsub(".", function(c) table.insert(t, c) end)
-
-        return t;
+        str = util.REQUIRES_IMPLEMENTATION_MESSAGE
     end
+
+    str:gsub(".", function(c) table.insert(t, c) end)
```

Comment:
Replace this line with `table.insert(t, str)` so Lua don't print each letter in new line!